### PR TITLE
feat(Operators): clean up and add equals operator, remove filter + do…

### PR DESCRIPTION
**Pull request by christianalfoni**
…cs updates

- Fixed some docs in general
- Removed `filter` operator in favor of only using `when` (allow it to take a function)
- Added `equals` operator to go down specific paths... it is beautiful
- Refactored operators internal to make more sense of using its API

### CODE REVIEW

File: packages/docs/content/api/05_operators.en.md [hunk 1]
```diff
@@ -5,113 +5,198 @@ title: Operators
 ## Operators
 Operators are function factories that returns an action. These allows you to do common things like updating state and control the flow.
 
+### Template tags
+When using operators you typically use a lot of template tags. These template tags are plain JavaScript which is used to target and extract data. The tree template tags available are:
+
+```js
+import {state, input} from 'cerebral/operators'
```
Comment (henri-hulski) [inline packages/docs/content/api/05_operators.en.md]: Should this here and below not be `import {set, state, input}`?

File: packages/cerebral/README.md [hunk 2]
```diff
@@ -14,140 +14,7 @@ A state controller with its own debugger
...
-
-### Operators
-Operators are available on:
-
-```js
-import {set, copy} from 'cerebral/operators'
-```
-
-Look in folder for available operators
+Go to [https://cerebral.github.io](https://cerebral.github.io)
```
Comment (christianalfoni) [inline packages/cerebral/README.md]: Jup!

**Comment by christianalfoni** [comment]
Fixed both :)

**Comment by Guria** [comment]
Is it still compatible with @gaspard PR about StateContainer computed support?

**Comment by christianalfoni** [comment]
Have not made any changes to `StateContainer` or `Computed` in this pull?

**Comment by Guria** [comment]
Sorry, my bad

**Pull request merged**